proto/api/v1/operations.proto

```diff
@@ -41,6 +41,12 @@ message ListOperationsRequest {
   // Default value is `created_at desc`
   // NOTE: On http protocol, this operates as a query parameter.
   string order_by = 2;
+  // Parameter to check the operations status.
+  optional string status = 3;
```

Comment:
@mflilian we must rename it to `stage` now, right? maybe this PR should be rebased with https://github.com/topfreegames/maestro/pull/507